2023-07-03 16:10:26,164: [11] m.main:INFO: Registered plugins: IntrospectionAPIPlugin, <class 'manticore.ethereum.plugins.SkipRevertBasicBlocks'>, <class 'manticore.ethereum.plugins.FilterFunctions'>
2023-07-03 16:10:26,165: [11] m.main:INFO: Beginning analysis
2023-07-03 16:10:26,168: [11] m.e.manticore:INFO: Starting symbolic create contract
2023-07-03 16:10:39,785: [11] m.e.manticore:INFO: Starting symbolic transaction: 0
2023-07-03 16:10:48,499: [11] m.e.manticore:INFO: 0 alive states, 4 terminated states
Traceback (most recent call last):
  File "/usr/local/bin/manticore", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/manticore/__main__.py", line 42, in main
    ethereum_main(args, logger)
  File "/usr/local/lib/python3.7/dist-packages/manticore/ethereum/cli.py", line 145, in ethereum_main
    m.finalize(only_alive_states=args.only_alive_testcases)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/manticore.py", line 177, in newFunction
    return func(self, *args, **kw)
  File "/usr/local/lib/python3.7/dist-packages/manticore/ethereum/manticore.py", line 1811, in finalize
    "{:x}: {:2.2f}%\n".format(int(address), self.global_coverage(address))
  File "/usr/local/lib/python3.7/dist-packages/manticore/ethereum/manticore.py", line 1888, in global_coverage
    runtime_bytecode = state.solve_one(code)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/state.py", line 483, in solve_one
    return self.solve_one_n(expr, constrain=constrain)[0]
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/state.py", line 494, in solve_one_n
    return self.solve_one_n_batched(exprs, constrain)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/state.py", line 509, in solve_one_n_batched
    values = self._solver.get_value_in_batch(self._constraints, exprs)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/smtlib/solver.py", line 745, in get_value_in_batch
    "Solver could not find a value for expression under current constraint set"
manticore.exceptions.SolverError: Solver could not find a value for expression under current constraint set
